### ISSUE TITLE
A channel separates

### ISSUE THREAD

**Issue by ScottMansfield** [go-proverbs/go-proverbs.github.io#5]
Channels provide separation physically and in go code. It's a way to separate goroutines that would otherwise have to know a lot about each other.

**Comment by elliott5** [comment]
How about rewording it to:

"Channels  orchestrate; orchestration simplifies; simplicity is prerequisite for reliability."

The last clause is from Edsger W. Dijkstra, How do we tell truths that might hurt? (1975).
